charts/victoria-logs-single/values.yaml

```
@@ -219,7 +219,9 @@ server:
     # -- Array of host objects
     hosts: []
     #   - name: vmselect.local
-    #     path: /select
+    #     path:
+    #       - /select
+    #       - /ui
```

Let's use `/insert` as an example as `/ui` does not exist and might confuse some users.
```suggestion
    #       - /insert
```